app/models/concerns/public_body_derived_fields.rb

```diff
@@ -8,12 +8,12 @@ module PublicBodyDerivedFields
 
     # When name or short name is changed, also change the url name
     def short_name=(short_name)
-      super
+      write_attribute("short_name", short_name)
```

Comment:
Don't understand this change without more context.

Should there be tests changed/added here?